### ISSUE TITLE
Add detailed analytics to the admin page

### ISSUE THREAD

**Issue by MichaelStatt-S** [FRC5892/HeroHours#13]
- [ ] filter by meeting
- [ ] sort by active ( attended a meeting in the last x)

**Comment by yyy2356** [comment]
this can still be done, but Trevor.io makes it easier and is technically done.

**Comment by MichaelStatt-S** [comment]
I don't know how you set up Trevor, but I don't really want to have to open that and run a query every day. I like the present  spreadsheet approach, but it is very clunky. Daily attendance should either be automated in the sheet or made into a view in the admin page

**Comment by yyy2356** [comment]
Trevor updates all existing queries and dashboards automatically to fit the data and sends that update to sheets, as i have done in our attendance sheet. currently, the only thing that needs to be done manually is make new queries for each individual meetings attendance, which is as simple as copying the previous one and changing the filtered date.

**Comment by MichaelStatt-S** [comment]
Yes, but I would like to remove that step of copying and pasting. That should be easily automated.

**Comment by MichaelStatt-S** [comment]
all the IMPORTDATA formula does is go to a web url that returns a csv. We could implement that ourselves and take in an input (meeting date) as url parameter. We could then still use the format I made, and we would only need to add months ahead of time. I would prefer to keep the look of my month-separated sheets (without the 200+ character formula)

**Comment by yyy2356** [comment]
its very possible that trevor already supports something like that, in which case that would be easier. ill look into it

**Comment by yyy2356** [comment]
also, making apis is annoying

**Comment by yyy2356** [comment]
look in the spreadsheet, i added a sheet called "attendance by meeting"
this should do what you want, given that the logs update hourly

**Comment by yyy2356** [comment]
can automate inserting new dates
